test/unit/services/fact_parser_test.rb

```diff
@@ -126,7 +132,7 @@ class DebianFactParserTest < ActiveSupport::TestCase
       as_admin do
         os = @facts_parser.operatingsystem
 
-        assert_equal '10', os.major
+        assert_equal '8', os.major
```

Comment:
Maybe we should change the test name as well - test no longer tries to parse facts for debian unstable.